pkg/splunk/enterprise/clustermaster_test.go

```diff
@@ -117,18 +117,18 @@ func TestGetClusterMasterStatefulSet(t *testing.T) {
+	test(splcommon.TestGetCMStatefulSet)
```

Comment:
Returned STS spec is the result of the config we are passing above, and there is not much we can leverage with this change(?). Any reason why you want to move it to a common place?

Same comment for other places.

Reply:
This is because in the response there are cases with biased language such as master/slave.